src/notificaties.md

```diff
@@ -0,0 +1,58 @@
+## Notificaties
+## Berichtkenmerken voor Open Klant API
+
+Kanalen worden typisch per component gedefinieerd. Producers versturen berichten op bepaalde kanalen,
+consumers ontvangen deze. Consumers abonneren zich via een notificatiecomponent (zoals <a href="https://notificaties-api.vng.cloud/api/v1/schema/" rel="nofollow">https://notificaties-api.vng.cloud/api/v1/schema/</a>) op berichten.
+
+Hieronder staan de kanalen beschreven die door deze component gebruikt worden, met de kenmerken bij elk bericht.
+
+De architectuur van de notificaties staat beschreven op <a href="https://github.com/VNG-Realisatie/notificaties-api" rel="nofollow">https://github.com/VNG-Realisatie/notificaties-api</a>.
+
+
+### internetaken
+
+**Kanaal**
+`internetaken`
+
+**Main resource**
+
+`internetaak`
+
+
+
+**Kenmerken**
+
+* `nummer`: Uniek identificerend nummer dat tijdens communicatie tussen mensen kan worden gebruikt om de specifieke interne taak aan te duiden.
+* `gevraagde_handeling`: Handeling die moet worden uitgevoerd om de taak af te ronden.
```

Comment:
Noticed this just now that this is in snake_case while the actual value key we send in the notification is in camelCase, this is probably a bug in the `generate_notificaties` command, it should show the keys in camelCase in this file as well. 

Can you create a separate issue for this in notifications-api-common? For now we can modify it manually to not delay this feature further.

```suggestion
* `gevraagdeHandeling`: Handeling die moet worden uitgevoerd om de taak af te ronden.
```